zio-http/src/test/scala/zio/http/endpoint/EndpointSpec.scala

```diff
@@ -511,6 +513,143 @@ object EndpointSpec extends ZIOSpecDefault {
           )
         },
       ),
+      suite("multipart/form-data")(
```

Comment:
This is one case where it could be useful to do a `Gen` spec:

 - generate N form fields
 - each form field generated from N types
 - form fields randomly have either a name, or no name
 - we generate round trip: decode multi-part, and encode multi-part

Otherwise the number of combinations is just staggeringly high and would be too painful to write by hand.